src/tusb_option.h

```diff
@@ -70,6 +70,8 @@
 
 #define OPT_MCU_CXD56         400 ///< SONY CXD56
 
+#define OPT_MCU_VALENTYUSB_EPTRI    500 ///< Fomu eptri config

```

Comment:
not a big deal, but it is best to change this to 600, since there is WIP PR for msp430 that use this value https://github.com/hathach/tinyusb/pull/194/files#diff-73307617739bffdbee43cd26855ee4c0R72